Source/WebKit/WebProcess/Plugins/PDF/PDFPlugin.mm

```diff
@@ -691,6 +687,9 @@ static void getAllScriptsInPDFDocument(CGPDFDocumentRef pdfDocument, Vector<Reta
     [m_containerLayer addSublayer:m_scrollCornerLayer.get()];
     if ([m_pdfLayerController respondsToSelector:@selector(setDeviceColorSpace:)])
         [m_pdfLayerController setDeviceColorSpace:screenColorSpace(m_frame->coreFrame()->view()).platformColorSpace()];
+    
+    if ([m_pdfLayerController respondsToSelector:@selector(setUseIOSurfaceForTiles:)])
+        [m_pdfLayerController setUseIOSurfaceForTiles:false];
```

Comment:
The method exists on the `Class` and not the instance – use either `getPDFLayerControllerClass()` or `[m_pdfLayerController class]`.

Reply:
Ah yeah right. Fixed.